src/modules/print/BasicsPrintView.js

```diff
@@ -167,7 +167,7 @@ class BasicsPrintView extends PrintView {
 
     this.renderCalendarSection({
       title: this.sections.basals.title,
-      data: this.aggregationsByDate.basals.byDate,
+      data: this.aggregationsByDate.basals.basal.byDate, // TODO: do we need to also add the suspend deviceEvents here?
```

Comment:
Has this question been answered somewhere?

Reply:
Ah, good catch.  I realized the answer is no, we're good with just passing the basal data in here.  Forgot I'd put this in.

Reply:
Will remove the comment and merge